tests/test_stroke.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+from wheatley.stroke import Stroke, HANDSTROKE, BACKSTROKE
+
+
+class BellTests(unittest.TestCase):
```

Comment:
```suggestion
class StrokeTests(unittest.TestCase):
```

Thanks for the Bell tests as well :+1:

Reply:
Oops! The copy and paste job has been rumbled!